### DISCUSSION TITLE
3.1.0rc3

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#26296] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 3.1.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/3.1.0rc3/

Git tag for the release:
https://github.com/apache/superset/tree/3.1.0rc3

The Change Log for the release:
https://github.com/apache/superset/blob/3.1.0rc3/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/3.1.0rc3/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/3.1.0rc3'>3.1.0rc3</a>.</em>

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26298

**Reply by john-bodley** [reply]
This is not considered a release blocker as it is not perceived as bug, but rather a fix in how non-numerics were implicitly cast.

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26317

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26262

**Reply by john-bodley** [reply]
This is not considered a release blocker as it has existed since 3.0.0 and it only affects a really specific user flow.

**Reply by michael-s-molina** [reply]
@kgabryje is working on a fix and we'll cherry-pick it in the next patch.

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26340

**Comment by glauberssoutodossantos** [comment]
+1

**Comment by VictorDMe** [comment]
+1

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26267

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26302

**Reply by michael-s-molina** [reply]
Even though a fix was submitted, we don't consider this a release blocker.

**Comment by sfirke** [comment]
I reopened #26254 as it persists in 3.1.0rc3.  It has changed behavior slightly, comparing my first post on that thread with my post just now for rc3 - let me know if this should instead be a new issue and I'll redo it.

**Reply by michael-s-molina** [reply]
Agreed that it's not a release blocker. Check [this](https://github.com/apache/superset/issues/26254#issuecomment-1863420328) for more context.

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26404

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/26323

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/26404

**Comment by wh474718572** [comment]
+1

**Comment by russell-koh** [comment]
Hi @michael-s-molina, i'm trying to install superset version 3.1.0 on AWS. Do you have a working version for AWS at 3.1.0 as the image that is used on aws currently is 3.0.0 (public.ecr.aws/p9r6s5p7/superset:latest)

**Reply by michael-s-molina** [reply]
Hi @russell-koh. We don't officially support AWS images currently. As defined by the [Superset Release Process](https://github.com/apache/superset/issues/26015), the following are officially supported:

> [Apache Website](https://downloads.apache.org/superset/)
> [PyPi](https://pypi.org/project/apache-superset/)
> [Docker](https://hub.docker.com/r/apache/superset)

Maybe these articles can help you:
- https://www.restack.io/docs/superset-knowledge-deploy-apache-superset-aws
- https://www.obytes.com/blog/superset-deployment

**Comment by wh474718572** [comment]
+1

**Comment by Yuval-Moshe** [comment]
+1

**Comment by fdmgit** [comment]
+1

**Comment by mboujaha** [comment]
+1

**Comment by LyleScott** [comment]
+1

**Comment by sundaram-dubey** [comment]
+1

**Comment by michael-s-molina** [comment]
Closing this as [RC4](https://github.com/apache/superset/discussions/26436) is out.
